scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtRunner.scala

```diff
@@ -80,7 +80,8 @@ object ScalafmtRunner {
       .withAllowTrailingCommas(true) // SIP-27, 2.12.2
     val scala213 = Scala213
       .withAllowTrailingCommas(true)
-    val default = scala213
+    val default = Scala213Source3
```

Comment:
please don't change the default.

also, you should add the newly supported dialects in the decoder below, otherwise they can't be used.

Reply:
Why would that be a problem? This is just a small subset of syntax changes and people would be super confused if they needed to change the dialect. Scala213 us basically the subset of Scala213Source3, so it only supports more not less.

Reply:
a dialect dictates how rewrite rules can be applied. so we can't use a superset, or the result might not compile.

Reply:
ok sure, I haven't thought about it.